nf_core/modules/bump_versions.py

```diff
@@ -0,0 +1,317 @@
+"""
+Bump versions for all modules on nf-core/modules
+or for a single module
+"""
+
+
+from __future__ import print_function
+import logging
+import questionary
+import re
+import rich
+from rich.console import Console
+from rich.table import Table
+from rich.markdown import Markdown
+import rich
+from nf_core.utils import rich_force_colors
+import sys
+
+import nf_core.utils
+import nf_core.modules.module_utils
+from nf_core.modules.nfcore_module import NFCoreModule
+from .modules_command import ModuleCommand
+
+log = logging.getLogger(__name__)
+
+
+class ModuleVersionBumper(ModuleCommand):
+    def __init__(self, pipeline_dir):
+        super().__init__(pipeline_dir)
+
```

Comment:
```suggestion
    self.up_to_date = None
    self.updated = None
    self.failed = None
    self.show_up_to_date = None
```
I think the code becomes clearer if all fields of the class are declared in the constructor, even though the execution of module's command always passes through `bump_versions` which would initialise them.